packages/web-components/package.json

```diff
@@ -90,9 +90,9 @@
     "webpack": "^4.43.0"
   },
   "dependencies": {
-    "@microsoft/fast-colors": "^5.1.0",
+    "@microsoft/fast-colors": "^5.1.3",
```

Comment:
@eljefe223 we can actually keep this at ^5.1.0 - that will pull in the patch versions.